doc/source/index.rst

```diff
@@ -36,6 +36,11 @@ arguments are to be passed for the initialization of the class.
 If the callable has a ``stop`` method, it will be called without arguments at the
 end of each run (one scene).
 
+If the configuration has a ``timeout``, that will be used as the maximum time
+in seconds the plugin will be allowed to run.  If it has not completed within
+this number of seconds, the job will be considered to have failed.  This feature
+is only available on Unix.
```

Comment:
Maybe "POSIX compliant" or "Linux" depending which is closer, we don't formally support Unix :wink:

Reply:
I just took what it says in the Python documentation for [`signal.setitimer`](https://docs.python.org/3/library/signal.html#signal.setitimer).  What does trollflow2 formally support?  If trollflow2 is Linux-only I can remove the line entirely.  Has anyone tried to run trollflow2 on anything else than Linux?

Reply:
trollflow is tested on linux and mac at the moment.

Reply:
Linux and Mac are both flavours of Unix, aren't they?

Reply:
Apparently my solution doesn't work on 3.7 for Mac, but does on 3.8 and 3.9.  Must I fix this, or can I skip this test for Mac on 3.7?  It's difficult for me to test on a Mac and I don't have access to one.  Or I can declare that this feature is supported only on Linux and skip the tests on Mac completely?

Reply:
I've changed the text now to state it is only supported on Linux, and I've disabled the test for other platforms.